js/base/Exchange.js

```diff
@@ -817,6 +817,10 @@ module.exports = class Exchange {
         return this.createOrder (symbol, 'limit', ...args)
     }
 
+    createTimeInForceOrder (symbol, ...args) {
```

Comment:
If I understand these types of orders right it looks like in case `timeInForce === 'gtc'` it should call default `createOrder`?

Reply:
If that is the default of the exchange, which is probably true for all exchanges